assets/js/screens/BatchEdit/BatchEdit.jsx

```diff
@@ -3,7 +3,608 @@ import Layout from "../Layout";
 import UIBreadcrumbs from "../../components/UI/Breadcrumbs";
 import BatchEditPreviewItems from "../../components/BatchEdit/PreviewItems";
 import BatchEditTabs from "../../components/BatchEdit/Tabs";
-
+const selectedItemsMock = [
```

Comment:
Can we extract this mock array to `assets/js/mock-data` and put in it's own file called something like "mock-batch-edit-data.js"?   So that file would look like:

```
export mockBatchEditData = [ { .....  } ]
```